virtcontainers/api.go

```diff
@@ -11,6 +11,7 @@ import (
 	"syscall"
 
 	deviceApi "github.com/kata-containers/runtime/virtcontainers/device/api"
+	"github.com/kata-containers/runtime/virtcontainers/device/config"
```

Comment:
You could follow the same thing you did with the above import:
```go
import (
	deviceConfig "github.com/kata-containers/runtime/virtcontainers/device/config"
)
```

Reply:
OK